src/components/media-views.js

```diff
@@ -156,16 +155,6 @@ function createVideoTexture(url, contentType) {
       } else if (videoEl.canPlayType(contentType)) {
         videoEl.src = url;
         videoEl.onerror = reject;
-
-        // HACK aframe iOS HLS video hacks
-        if (isIOS) {
```

Comment:
Looks like these hacks only break things, and things seem fine without them or any sort of special shader. Chalk it up to iOS updates?

Reply:
down with the HACK